nexus/types/src/deployment/blueprint_diff.rs

```diff
@@ -58,6 +58,9 @@ impl<'a> BlueprintDiffSummary<'a> {
         let BlueprintDiff {
             // Fields in which changes are meaningful.
             sleds,
+            // TODO Will need to diff these when we can actually create
+            // blueprints with pending MGS updates.
```

Comment:
This seems kinda dicey / easy to forget. This method is only returning true or false - can we go ahead and do that the same way we check `sleds` below?

Reply:
Done in 244c42b4b2c750f73e355a614b00e64e3f126874.